sorc/ncep_post.fd/MDL2P.f

```diff
@@ -501,12 +500,12 @@ SUBROUTINE MDL2P(iostatusD3D)
                  IF(CFR(I,J,LL) < SPVAL .AND. CFR(I,J,LL-1) < SPVAL)          &
                    CFRSL(I,J) = CFR(I,J,LL) + (CFR(I,J,LL)-CFR(I,J,LL-1))*FACT 
 ! DUST
-                 if (gocart_on) then
-                   DO K = 1, NBIN_DU
-                     IF(DUST(I,J,LL,K) < SPVAL .AND. DUST(I,J,LL-1,K) < SPVAL)   &
-                     DUSTSL(I,J,K) = DUST(I,J,LL,K) + (DUST(I,J,LL,K)-DUST(I,J,LL-1,K))*FACT
-                   ENDDO
-                 endif
+!                 if (gocart_on) then
```

Comment:
@zhanglikate Since these are legacy code, could they be deleted instead of commented out?

Reply:
@WenMeng-NOAA  Removed. Thanks.